plugin/draft_editor/draft_editor.py

```diff
@@ -104,15 +104,26 @@ def get_editor_app() -> ui.App:
     raise RuntimeError("Draft editor is not running")
 
 
-def close_editor(submit_draft: bool):
-    global last_draft
+def close_editor(submit_draft: bool) -> None:
     remove_tag("user.draft_editor_active")
     actions.edit.select_all()
     selected_text = actions.edit.selected_text()
```

Comment:
I would just replace this line with
```
if submit_draft:
        last_draft = actions.edit.selected_text()
```
Then we store the selected text as early as possible in case of anything else later throwing an exception

Reply:
makes sense, fixed